src/components/profile/edit/EditPersonalInfo.vue

```diff
@@ -419,7 +420,7 @@ export default {
     initialValues: Object,
   },
   components: {
-    Combobox,
+    Combobox: () => import('@/components/ui/Combobox.vue'),
```

Comment:
why is this loaded lazily, although (at least afaic) it is loaded everytime?

Reply:
I'm starting to chunk up the dependencies. This produces multiple chunks in a `npm run build` (check it out). See https://vuejs.org/v2/guide/components-dynamic-async.html
The goal is to lazy load them at some point. So theses components are only loaded when a user edits their profile.

Reply:
Ah, I see. Weird semantic, though